src/Model/Contact.php

```diff
@@ -1878,7 +1877,8 @@ public static function updateFromProbe($id, $network = '', $force = false)
 		self::updateContact($id, $uid, $ret['url'], $ret);
 
 		// Update the corresponding gcontact entry
-		PortableContact::lastUpdated($ret["url"]);
+//		PortableContact::lastUpdated($ret["url"]);
```

Comment:
Why the comment?

Reply:
Because I forgot to remove that line :grin:

Reply:
In fact I'm currently working at some GContact stuff and got drifted away. I think you know that problem :grimacing: 

So I hold back my changes in the gcontact file and wanted to commit this thing here first.